### PR TITLE
Add twitter bootstrap to CakePHP

### PR DESCRIPTION
**Pull request by flangofas**
Updated:
- CakePHP 3.0.8
- jQuery 2.1.4
- Bootstrap 3.3.5
- CakePHP Bootstrap Helpers: https://holt59.github.io/cakephp3-bootstrap3-helpers/

### CODE REVIEW

File: src/Template/Layout/default.ctp [hunk 1]
```diff
@@ -1,59 +1,42 @@
-<?php
```
Comment (mamchenkov) [inline src/Template/Layout/default.ctp]: @flangofas the change here shows not only content change, but also permission change from 644 (rw-r--r--) to 755 (rwx-r-xr-x).  We don't need executable permissions on the file.

File: src/Controller/AppController.php [hunk 2]
```diff
@@ -37,5 +53,6 @@ class AppController extends Controller
     public function initialize()
     {
         $this->loadComponent('Flash');
+        $this->layout = 'default';
```
Comment (mamchenkov) [inline src/Controller/AppController.php]: @flangofas what's happening if you DON'T do this?  I thought the default layout was pretty much assumed.

File: src/Template/Layout/default.ctp (hunk 1 above)
Comment (flangofas) [inline src/Template/Layout/default.ctp]: @mamchenkov done!

File: src/Controller/AppController.php (hunk 2 above)
Comment (flangofas) [inline src/Controller/AppController.php]: @mamchenkov good catch. :+1:

**Comment by mamchenkov** [comment]
@flangofas  Excellent work! Thanks :+1:

**Pull request merged**